src/display/editor/tools.js

```diff
@@ -611,15 +611,27 @@ class AnnotationEditorUIManager {
       );
     };
 
+    const textInputChecker = (_self, { target: el }) => {
```

Comment:
Given that `FreeText` has its own `KeyboardManager`-instance, do we need to add the same kind of check for any of those shortcuts or is that fine as-is?

https://github.com/mozilla/pdf.js/blob/f95a2bbe2944d63fc39916113c617339194dcbf1/src/display/editor/freetext.js#L65

Reply:
It should be fine since when we're editing a freetext, we've to move the focus on an input and then the editing session will be stopped.